integration/environment/machine.go

```diff
@@ -45,7 +45,7 @@ func (m *Machine) CreatePod(namespace string, pod corev1.Pod) (TearDownFunc, err
 	client := m.Clientset.CoreV1().Pods(namespace)
 	_, err := client.Create(&pod)
 	return func() error {
-		err := client.Delete(pod.GetName(), &metav1.DeleteOptions{})
+		err := client.Delete(pod.GetName(), metav1.NewDeleteOptions(0))
```

Comment:
Why do you think `grace period = 0` is required?

Reply:
I found many integartion-test cases failed due to `timed out waiting for the condition` in and found that pod stuck in `Terminating`.

Reply:
OK. After #165 merged, will remove this commit.

Reply:
#165 is now merged

Reply:
Done